src/components/TweetCreationForm.tsx

```diff
@@ -1,7 +1,14 @@
 import Picker, { IEmojiData } from "emoji-picker-react";
-import { memo, MouseEvent, useRef, useState } from "react";
+import {
+  createRef,
+  memo,
+  MouseEvent,
+  RefObject,
+  useEffect,
+  useState,
+} from "react";
 import Avatar from "react-avatar";
```

Comment:
Delete this library from project, use avatar from redux
Example: 
```
  const avatar = useSelector<RootState, RootState["user"]["profile"]["avatar"]>(
    (store) => store.user.profile.avatar
  );

<img src={avatar}  />

```

Reply:
this has already been implemented, just the extra library has not been removed. I will do

Reply:
use <img  src={avatar} /> instead of <Avatar />. Because the library that imports <Avatar /> will be deleted